src/Microsoft.Health.Dicom.Core.UnitTests/Features/Query/QueryParserTests.cs

```diff
@@ -33,6 +34,7 @@ public QueryParserTests()
         [InlineData("includefield", "StudyDate, StudyTime")]
         public void GivenIncludeField_WithValidAttributeId_CheckIncludeFields(string key, string value)
         {
+            EnsureArg.IsNotNull(value, nameof(value));
```

Comment:
Generally for strings we do `IsNotNullOrWhiteSpace`. Also, any reason to not do the key?

Reply:
I did the ones the editor warned about. Normally this was due to accessing a member, EX: value.length . And that would only lead to checking one of the values. Now in this case I do not know why the editor warned about key and not value.

Reply:
resolving since tests